images/ci/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
```

Comment:
How about using Go 1.16? I've seen it in other places...

Reply:
oh good catch, indeed :+1: